benchmarks/bench_plot_fastkmeans.py

```diff
@@ -10,7 +10,7 @@
 def compute_bench(samples_range, features_range):
 
     it = 0
-    results = defaultdict(lambda: [])
```

Comment:
This file is just here for documentation of the experiment I ran. I can revert the diff once the PR has been reviewed.